server/webapp/WEB-INF/rails/lib/tasks/gen_webpack_js_routes.rake

```diff
@@ -39,4 +39,37 @@ task generated_js: :environment do
     f.puts("// This file is automatically generated by `generated_js` task. Any changes will be lost")
     f.puts TsRoutes.generate(include: JsRoutes.configuration.include)
   end
+
+
+  open(Rails.root.join("#{ENV['OUTPUT_DIR']}/gocd_version.ts"), 'w') do |f|
```

Comment:
This is where the typescript helper is at. Exposes identical data as in the `CurrentGoCDVersion` class.